### DISCUSSION TITLE
SILAC quantification using multiple labels

### DISCUSSION THREAD

**Discussion by brucewayne949** [vdemichev/DiaNN#1131] [General]
Hi Vadim and everyone

I was trying to quantify a DIA dataset which contains +4 and +8 heavy labelled lysines only (and not arginines). Based off previous SILAC discussions, I thought this is how the the 'additional options' should look like, but I'm not sure. 

--reanalyse
--smart-profiling 
--fixed-mod SILAC,0,K,label   
--lib-fixed-mod SILAC  
--channels SILAC,L,K,4.00078; SILAC,H,K,8.05021  
--peak-translation  
--original-mod

Is it possible to quantify data containing more than one heavy label using DIA-NN?

**Comment by vdemichev** [comment]
Hi,

Sounds correct, should work, but also please see the description of multiplexed data analysis here: https://github.com/vdemichev/DiaNN?tab=readme-ov-file#multiplexing-using-plexdia

Best,
Vadim

**Reply by vdemichev** [reply]
Mass on H lysine seems incorrect though

**Comment by brucewayne949** [comment]
Thanks Vadim!
